egs/wsj/s5/steps/dur_model/make_nnet_config.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+learning_rate=0.001
+natural_gradient=false
```

Comment:
No, I was not sure. So I will set it to true. I am currently experimenting it but the result is not ready yet.